docker/build_graphstorm_image.sh

```diff
@@ -78,6 +79,9 @@ parse_params() {
             SUFFIX="${2-}"
             shift
             ;;
+        -u | --use-parmetis)
```

Comment:
Maybe -up? u if just for use.

Reply:
Hmm, maybe I'll leave this as only the fully spelled out `--use-parmetis` to avoid having to remember too many single-letter flags.